gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/JavaServiceDistributionPlugin.java

```diff
@@ -332,4 +338,15 @@ private static void replaceManifestClasspath(Path windowsScript, String manifest
 
         Files.writeString(windowsScript, cleanedText);
     }
+
+    /** To make our unit-test setup simpler, we allow hard-coding a specific go-java-launcher version. */
+    private static String getGoJavaLauncherCoordinate(Project project, String coordinate) {
+        if (!project.hasProperty(TEST_GO_JAVA_LAUNCHER_VERSION_OVERRIDE_PROP)) {
```

Comment:
The name of this confused me a bit - thought we were overriding the version set by GCV at first. Maybe it makes sense to call this `TEST_GO_JAVA_LAUNCHER_FALLBACK_VERSION_OVERRIDE`? To make it obvious that it's about the fallback version.

Reply:
Makes sense, updated.